lms/static/sass/course/modules/_calculator.scss

```diff
@@ -129,13 +134,15 @@ div.calc-main {
           padding: 10px;
           -webkit-appearance: none;
           width: 100%;
+          background: $black;
```

Comment:
Please remove, this is making the input field display wrong, see screengrab below.

![image](https://cloud.githubusercontent.com/assets/163763/9253747/9082ab84-41ad-11e5-99d0-8d1e75d9211a.png)